astropy/io/fits/convenience.py

```diff
@@ -551,10 +551,10 @@ def table_to_hdu(table, character_as_bytes=False):
                         "though one has to enable the unit before reading.")
                 else:
                     warning += (
-                        "and cannot be recovered in reading. If pyyaml is "
-                        "installed, it can roundtrip within astropy by "
-                        "using QTable both to write and read back, "
-                        "though one has to enable the unit before reading.")
+                        "and cannot be recovered in reading. Using pyyaml it "
```

Comment:
Maybe remove "using pyyaml"? It now becomes an implementation detail of astropy.

Reply:
Done!